src/server/auth.ts

```diff
@@ -74,3 +75,50 @@ export const getServerAuthSession = (ctx: {
 }) => {
   return getServerSession(ctx.req, ctx.res, authOptions);
 };
+
+
+/*
+todo write docs
+*/
+export interface session {
+  account : Number;
+  token : String;
+  expiry : Date;
+}
+
+export const validateUser = async (
+  id:number, 
+  token:string
+  ) => {
+
+    const result = await prisma.session
+    .findMany({where: 
+            { 
+              account: id,
+              token : token 
+            } } )
+    .catch(message => {
+              throw new TRPCError({
+                 code: "INTERNAL_SERVER_ERROR",
+                 message,
+              })})
+    
+   const now = new Date();
+
+    if(!result[0])
+    {
+      throw new Error("Session does not currently exist");
```

Comment:
```suggestion
      console.error("Didn't find session in", result[0]);
      throw new Error("Session does not currently exist");
```

I'm not sure what the type of `result[0]` is, but if it's an object, `console.error` will display it nicely.

If it's a primitive, we can put it in the `new Error`.

@meows might have a smarter approach to logging.